lib/galaxy/datatypes/binary.py

```diff
@@ -126,6 +126,35 @@ def sniff(self, filename):
             return False
 
 
+class Gpr(Binary):
+
+    """Binary data in GPR, Axon Text File (ATF) format."""
```

Comment:
For sure, where? in the class or the data types config file?

Reply:
I meant in the class comment, something like:
```python
    """
    GenePix Results format, described at
    http://mdc.custhelp.com/app/answers/detail/a_id/18883#gpr
    """
```

> I put it in binary because it's the same as cell, both describe a microarray experiment result.

The `cel` is format is clearly binary, see e.g. `lib/galaxy/datatypes/test/test.cel`, while `gpr` is not. So, you can put the `Gpr` class either together with its parent class (which, as suggested above, should be `Tabular` or `Text`) or in a separate file for microarray datatypes.